app/liquid/views/partials/_recommend_pages.liquid

```diff
@@ -7,11 +7,9 @@
 
 <script type="text/javascript">
   $(function() {
-    var global = window.champaign.personalization;
-    var data = {
+    window.event.emit('champaign:recommend_pages:init', 'recommend-pages-component', {
       locale: '{{locale}}',
       pageId: "{{ id }}"
-    };
-   mountRecommendPages('recommend-pages-component', data);
```

Comment:
Does this mean we're no longer mounting components by calling a function? we're now emitting an event?

Reply:
I think overall this is a cleaner approach that's less error prone, but I've only updated recommend pages for now as the feature is the newest and meant the least effort. Using events we avoid throwing errors in the template that may break the entire page.

I also refactored the `RecommendPages` component as it didn't need its own pack (so I've added it to member_facing.js) and it's also less error prone.